src/Rules/Resources/HowToFix.resx

```diff
@@ -150,51 +150,51 @@
   </data>
   <data name="IsContentElementFalseOptional" xml:space="preserve">
     <value>Set the IsContentElement UI Automation property to false.</value>
```

Comment:
@codeofdusk, should "UI Automation" be locked in these strings? It's currently localized